web/environments.go

```diff
@@ -38,9 +37,9 @@ func NewEnvironmentListHandler(client consul.Client) gin.HandlerFunc {
 	}
 }
 
-func NewLandscapesListHandler(client consul.Client) gin.HandlerFunc {
+func NewLandscapeListHandler(client consul.Client) gin.HandlerFunc {
```

Comment:
It is not exactly about this line, but we still have `NewEnvironmentsListHandler` above. The follow the same convention, it should be `NewEnvironmentListHandler` or this one `NewLandscapesListHandler, isn't it?

Below we have `NewSAPSystemsListHandler` with a `s` too